packages/frontend/src/lib/Tooltip.svelte

```diff
@@ -58,14 +58,15 @@ export let bottom = false;
 export let bottomLeft = false;
 export let bottomRight = false;
 export let left = false;
+export let tipClass = '';
 </script>
 
 <div class="relative inline-block">
   <span class="group tooltip-slot {$$props.class}">
     <slot name="content" />
   </span>
   <div
-    class="whitespace-nowrap absolute tooltip opacity-0 inline-block transition-opacity duration-150 ease-in-out pointer-events-none z-[10]"
+    class="whitespace-nowrap absolute tooltip opacity-0 inline-block transition-opacity duration-150 ease-in-out pointer-events-none z-[10] {tipClass}"
```

Comment:
Missing rebase ?

Reply:
Nice catch. Removed. Thanks!!

Reply:
I actually rebased but i forgot to remove that part